### PR TITLE
Ghassan cv

### PR DESCRIPTION
**Pull request by Saadat1234**
Ghassan Cv in index html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+</head>
+
+<body>
+  <header>
+    <h1>Curriculum vitae</h1>
+  </header>
+  <main>
+    <section>
+      <h2>Objective</h2>
+      <p>To work with an organization where I can utilize my acquired skills in the best possible ways for the best interests of the organization and facilitate the organization/institution in achieving its goals.</p>
```
Comment (MrHezekiah) [inline index.html]: The length of your codes here need shortening so it fits the viewport width, without having to drag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    </section>
+    <!-- ///////////////////////////////////-->
+    <section>
+      <h2>WORK EXPERIENCE</h2>
+      <section>
+        <h3>Taxi Driver</h3>
+        <!-- Name of position  -->
+        <p>
+          <strong>Duration:</strong> Nov-2010 to Feb 2012
```
Comment (MrHezekiah) [inline index.html]: You should use the <time> element and the "datetime" attribute in representing dates and time in a machine readable format.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        </p>
+        <!-- title and Duration -->
+      </section>
+      <section>
+        <h3>Pizza deliver</h3>
+        <p><strong>Duration: </strong>21 March 2013- 30 Sep 2015</p>
```
Comment (MrHezekiah) [inline index.html]: Same as above; use the <time> element and the "datetime" attribute in representing dates and time in a machine readable format.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      </section>
+      <section>
+        <h3>Bus Driver</h3>
+        <p>
+          <strong>Duration: </strong>May 2015- Feb 30th 2016
```
Comment (MrHezekiah) [inline index.html]: Same as above; use the <time> element and the "datetime" attribute in representing dates and time in a machine readable format.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        </p>
+      </section>
+    </section>
+    <!-- ///////////////////////////////////-->
+    <section>
+      <h2>EDUCATIONAL BACKGROUND:</h2>
+      <p>2007-2012</p>
```
Comment (MrHezekiah) [inline index.html]: Same as above; use the <time> element and the "datetime" attribute in representing dates and time in a machine readable format.

File: index.html (hunk 1 above)
Comment (Saadat1234) [reply index.html]: thanks to reply me its not work because the html beautify is installed.

File: index.html (hunk 2 above)
Comment (Saadat1234) [reply index.html]: i am done with date and time

File: index.html (hunk 3 above)
Comment (Saadat1234) [reply index.html]: i am done with date and time

File: index.html (hunk 4 above)
Comment (Saadat1234) [reply index.html]: i am done with date and time

File: index.html (hunk 5 above)
Comment (Saadat1234) [reply index.html]: i am done with date and time

**Pull request merged**